modules/nf-core/basicpy/main.nf

```diff
@@ -0,0 +1,35 @@
+process BASICPY {
+    tag "$meta.id"
+    label 'process_single'
+
+    // Exit if running this module with -profile conda / -profile mamba
+    if (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1) {
+        exit 1, "Basicpy module does not support Conda. Please use Docker / Singularity instead."
+    }
+
+    container "yfukai/basicpy-docker-mcmicro:0.1.2"
+
+    input:
+    tuple val(meta), path(image)
+    val(cpu_gpu)
```

Comment:
Is this mandatory?

Reply:
Yes, chosing cpu or gpu is a mandatory positional argument for how this container is built. I will add this note into the meta.yml!

Reply:
Perfect! Thanks!